code-challenges.js

```diff
@@ -18,6 +46,12 @@ var cubeAndSum1 = [2, 3, 4]
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
+let sumedCubes = [];
+   for ( var i = 0; i < sumedCubes.length;i++) {
+      sumedCubes[i] = sumedCubes[i]*sumedCubes[i]*sumedCubes[i];
+   }
+
+console.log(sumedCubes(cubeAndSum1));
 
```

Comment:
The logic looks great. I would encourage you to think about using map if your input data is an array and your output data is an array. It will make your code cleaner, shorter, and more efficient.